Core/Core/SwiftUIViews/CoreHostingController.swift

```diff
@@ -18,61 +18,82 @@
 
 import SwiftUI
 
-public class CoreHostingController<InnerContent: View>: UIHostingController<CoreHostingBaseView<InnerContent>> {
-    public var navBarStyle = NavBarStyle.global
+public class CoreHostingController<Content: View>: UIHostingController<CoreHostingBaseView<Content>> {
+    public var navigationBarStyle = UINavigationBar.Style.global
```

Comment:
One thing I just thought of is perhaps we want the default to be `.modal` since that is used far more often than `.global`, or `.color(nil)` which would end up not trying to apply anything until a color or different case was explicitly given.

Reply:
Yeah that sounds good to me. `.global` _sounds_ like the right default but modal is probably the more practical default.